README.md

```diff
@@ -189,6 +189,11 @@ The following CMake options are available for adding/removing project features.
 <td>This option can be set to any of the next values:<br>- No_value: default value is used and no_platform library is used.<br>- "POSIX": Provides implementation for Linux and Mac systems.<br>- "WIN32": Provides platform implementation for Windows based system<br>- "USER": Tells cmake to use an specific implementation provided by user. When setting this option, user must provide an implementation library and set option `AZ_USER_PLATFORM_IMPL_NAME` with the name of the library (i.e. <code>-DAZ_PLATFORM_IMPL=USER -DAZ_USER_PLATFORM_IMPL_NAME=user_platform_lib</code>). cmake will look for this library to link az_core</td>
 <td>No_value</td>
 </tr>
+<tr>
+<td>STATIC_CRT</td>
+<td>This option has effect only when MSVC is used. It specifies, whether the SDK should be built with the static (ON) or the dynamic (OFF) C runtime dependency.</td>
```

Comment:
```suggestion
<td>This option has affect only when the MSVC compiler is used, specifically on Windows. It specifies whether the SDK should be built with the static (ON) or the dynamic (OFF) C runtime dependency.</td>
```

Reply:
Affect is verb, effect is noun. "Affects something", but "has effect", so it would need to be something like "Affects compilation when MSVC compiler is used".
I agree to "the MSVC compiler", but we don't need the "specifically on Windows" part. MSVC is Windows-only.

Reply:
Our support matrix is generally in terms of supported OSes (which dictate the default compiler and version). I wouldn't expect users to know what MSVC is and that MSVC is a Windows only compiler. 

What we want to convey is: If I am not running on Windows, this option is not useful.

> Affect is verb, effect is noun. "Affects something", but "has effect"

You are probably right. Updated. I used the short-hand if "result" can be substituted in for effect, in this case, and it couldn't.

Reply:
>I wouldn't expect users to know what MSVC is and that MSVC is a Windows only compiler.
So, you read "This option has effect only when the MSVC compiler is used", and you don't know what it is, then you are not using it, so you can skip reading the rest, the option does not apply to you.

> What we want to convey is: If I am not running on Windows, this option is not useful.
If you are using Intel compiler, for example, the option does not apply either.

Reply:
Regarding the comma, I don't think it is needed, just look how many occurrences are not using a comma there: https://www.bing.com/search?q=%22it+specifies%2C+whether%22&form=QBLH&sp=-1&pq=%22it+specifies%2C+whether%22&sc=0-23&qs=n&sk=&cvid=BD9996618C914542A4640B89E508E584

Reply:
> I don't think it is needed, just look how many occurrences are not using a comma there:


I think we both are saying the same thing :)

Reply:
ICC, supports C99, works on Windows and MacOS.